TranslatorReasonerAPI.yaml

```diff
@@ -349,7 +360,9 @@ components:
         - object
     Node:
       type: object
-      description: A node in the knowledge graph
+      description: >-
+        A node in the knowledge graph which represents some biomedical
+        concept.
```

Comment:
It may be helpful to add in the description that the nodes are identified by the keys in the KnowldgeGraph Node mapping.

Reply:
I don't know what this is referring to. I don't see mention of a node mapping in the KnowledgeGraph object here.

Reply:
I'm referring to the `nodes` property of `KnowledgeGraph`. Someone looking at the definition of `Node` may wonder where the CURIE should be specified. I think it would be helpful to mention that the CURIE is the key in the `nodes` dictionary / map.

Reply:
I was confused because `nodes` definition says that it's a "list of nodes in the KnowledgeGraph." Are you saying that this is a list of CURIE strings, or a dictionary of CURIE string: Node? If so, should I go ahead and change the definition of `node ` in this PR?